content/instructor/communication/next-steps/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+date: 2019-03-21
+title: You’ve published your first lesson, now what?
+description: 'Now that you have completed the full process of lesson creation, lets really nail it and get more practice in.'
+categories: ['instructor', 'communication']
+published: false
+---
+
+
+The first 30-second demo started with you uploading a clip to slack. Those days are past us now. Feel free to upload directly to egghead (to your hearts desire!) for the following lessons.
+
+The process will largely be the same as the first lesson you produced. When you upload a lesson, you can link to it in your personal instructor chat and it will be reviewed and iterated on with a coach.
+
+You'll notice each lesson will get a little easier to record and publishing will come a little faster.
+ 
+Thats progress. 🙌
+
+After 12 published lessons, you'll have the ability to self approve/publish your own lessons. We'll always be here for you if you'd like a review on a particular difficult lesson but at this point, we trust your judgement on producting lessons.
```

Comment:
```suggestion
For now, we want to focus on publishing single one-off lessons, but once you've published a handful of those, we can start to discuss your first course.

After you've published 12 lessons, you'll gain the ability to self approve/publish your own lessons. This is a big step, and by that point, you will feel comfortable and confident, but we are here to help no matter how many lessons you've published.
```

Reply:
github wouldn't let me commit this suggestion, did it manually